satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,26 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		user_id
+
+	field	user_id user.id	cascade
+	field	offer_id offer.id	cascade
+	field	referred_by user.id	cascade
+	field	credits_earned	int
+	field	credits_used	int
+	field	credits_type	text
```

Comment:
This type should be `int` and we will create the `const` in the code for the value